files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -69,14 +69,22 @@ destinations:
     runner: pulsar_embedded
     env:
       LC_ALL: C
+      GALAXY_MEMORY_MB: "{mem}"
+      GALAXY_SLOTS: "{cores}"
+      SINGULARITY_CACHEDIR: /data/share/var/database/container_cache
```

Comment:
This is on the pulsar side and an NFS share, correct?
Maybe we can add a comment here.

Reply:
yes
```suggestion
      SINGULARITY_CACHEDIR: /data/share/var/database/container_cache # On the NFS share on remote Pulsar side
```